tools/myst/host/package.c

```diff
@@ -864,6 +875,7 @@ int _exec_package(
     */
     options.nobrk = parsed_data.no_brk ? true : false;
     options.exec_stack = parsed_data.exec_stack ? true : false;
+    options.host_uds = parsed_data.host_uds ? true : false;
```

Comment:
Would use direct assignment (for all three lines). The source and destination are both ``bool``.